src/lib/one-blink-api.ts

```diff
@@ -48,6 +51,24 @@ export default class OneBlinkAPI {
     this.secretKey = secretKey
   }
 
+  /** @internal */
+  get oneBlinkUploader(): OneBlinkUploader {
+    if (this._oneBlinkUploader) {
+      return this._oneBlinkUploader
+    }
+    const uploader = new OneBlinkUploader({
+      apiOrigin: OneBlinkAPI.tenant.apiOrigin,
+      region: OneBlinkAPI.tenant.awsRegion,
+      getIdToken: () => {
+        return Promise.resolve(
+          generateJWT(this.accessKey, this.secretKey, this.jwtExpiry),
+        )
+      },
+    })
```

Comment:
Prob cleaner to just set this in the constructor.

Reply:
Agree with Natt